aws-test/tests/aws_ec2_gateway_load_balancer/test-list-query.sql

```diff
@@ -0,0 +1,3 @@
+select name, arn
+from aws_new.aws_ec2_gateway_load_balancer
+where name = '{{resourceName}}'
```

Comment:
Use any unique properties other than `name`, when run a query to verify list call
```suggestion
where arn = '{{ output.resource_aka.value }}';
```